### PR TITLE
add case_ids to scenario outline tests

### PR DESCRIPTION
**Pull request by pruan-rht**
similar to https://github.com/openshift/cucushift/pull/9207 

@liangxia, @jhou1 PTAL.

### CODE REVIEW

File: features/cli/limits.feature [hunk 1]
```diff
@@ -87,8 +87,8 @@ Feature: limit range related scenarios:
     @network-ovnkubernetes @network-openshiftsdn
     @heterogeneous @arm64 @amd64
     Examples:
-      | path | expr1 | expr2 | expr3 | expr4 | expr5 | expr6 | expr7 | expr8 | expr9 | expr10 |
-      | ocp12286 | 2Gi | min | 2Gi | max | 1Gi | 400m | min | 400m | max | 200m |
+      | case_id   | path     | expr1 | expr2 | expr3 | expr4 | expr5 | expr6 | expr7 | expr8 | expr9 | expr10 |
+      | OCP-12286 | ocp12286 | 2Gi   | min   | 2Gi   | max   | 1Gi   | 400m  | min   | 400m  | max   | 200m   |
```
Comment (liangxia) [inline features/cli/limits.feature:91]: Unexpected change ?

File: features/networking/ovn.feature [hunk 2]
```diff
@@ -439,13 +439,13 @@ Feature: OVN related networking scenarios
...
-      | HUP    |
-      | SEGV   |
-      | 9      |
+      | case_id   | signal |
+      | OCP-26091 | TERM   |
+      | OCP-26091 | QUIT   |
+      | OCP-26091 | INT    |
+      | OCP-26091 | HUP    |
+      | OCP-26091 | SEGV   |
+      | OCP-26091 | 9      |
```
Comment (liangxia) [inline features/networking/ovn.feature:448]: Unexpected change

File: features/networking/sriov.feature [hunk 3]
```diff
@@ -517,11 +517,11 @@ Feature: Sriov related scenarios
...
-      | file              | Error message                |
-      | invalidname.yaml  | invalid characters           |
-      | non-deviceid      | not supported                |
-      | non-vondor        | vendor 15b4 is not supported |
-      | vfnum0            | numVfs(0) in CR              |
+      | case_id   | file             | Error message                |
+      | OCP-25844 | invalidname.yaml | invalid characters           |
+      | OCP-25844 | non-deviceid     | not supported                |
+      | OCP-25844 | non-vondor       | vendor 15b4 is not supported |
+      | OCP-25844 | vfnum0           | numVfs(0) in CR              |
```
Comment (liangxia) [inline features/networking/sriov.feature:524]: Unexpected change

File: features/cli/limits.feature (hunk 1 above)
Comment (pruan-rht) [reply features/cli/limits.feature:91]: this is alignment change and added the case id

File: features/online/build.feature [hunk 4]
```diff
@@ -57,23 +57,23 @@ Feature: ONLY ONLINE related feature's scripts in this file
...
+      | OCP-9993 | jboss-webserver30-tomcat7-openshift | 1.2       | https://github.com/jboss-openshift/openshift-quickstarts.git | tomcat-websocket-chat |
+      | OCP-9993 | jboss-webserver30-tomcat7-openshift | 1.3       | https://github.com/jboss-openshift/openshift-quickstarts.git | tomcat-websocket-chat |
+      | OCP-9993 | jboss-webserver30-tomcat8-openshift | 1.1       | https://github.com/jboss-openshift/openshift-quickstarts.git | tomcat-websocket-chat |
+      | OCP-9993 | jboss-webserver30-tomcat8-openshift | 1.2       | https://github.com/jboss-openshift/openshift-quickstarts.git | tomcat-websocket-chat |
+      | OCP-9993 | jboss-webserver30-tomcat8-openshift | 1.3       | https://github.com/jboss-openshift/openshift-quickstarts.git | tomcat-websocket-chat |
+      | OCP-9993 | jboss-webserver31-tomcat7-openshift | 1.0       | https://github.com/jboss-openshift/openshift-quickstarts.git | tomcat-websocket-chat |
+      | OCP-9993 | jboss-webserver31-tomcat8-openshift | 1.0       | https://github.com/jboss-openshift/openshift-quickstarts.git | tomcat-websocket-chat |
```
Comment (liangxia) [inline features/online/build.feature:68]: Unexpected change

File: features/online/build.feature [hunk 5]
```diff
@@ -57,23 +57,23 @@ Feature: ONLY ONLINE related feature's scripts in this file
...
-      | image_name | image_tag | source_url                                            | context_dir |
-      | wildfly    | 8.1       | https://github.com/openshift/openshift-jee-sample.git | /           |
-      | wildfly    | 9.0       | https://github.com/openshift/openshift-jee-sample.git | /           |
-      | wildfly    | 10.0      | https://github.com/openshift/openshift-jee-sample.git | /           |
-      | wildfly    | 10.1      | https://github.com/openshift/openshift-jee-sample.git | /           |
+      | case_id  | image_name | image_tag | source_url                                            | context_dir |
+      | OCP-9997 | wildfly    | 8.1       | https://github.com/openshift/openshift-jee-sample.git | /           |
+      | OCP-9997 | wildfly    | 9.0       | https://github.com/openshift/openshift-jee-sample.git | /           |
+      | OCP-9997 | wildfly    | 10.0      | https://github.com/openshift/openshift-jee-sample.git | /           |
+      | OCP-9997 | wildfly    | 10.1      | https://github.com/openshift/openshift-jee-sample.git | /           |
```
Comment (liangxia) [inline features/online/build.feature:76]: Unexpected change

File: features/networking/ovn.feature (hunk 2 above)
Comment (pruan-rht) [reply features/networking/ovn.feature:448]: Added case id....

File: features/online/build.feature (hunk 5 above)
Comment (pruan-rht) [reply features/online/build.feature:76]: added case id for scenario outline

File: features/online/build.feature (hunk 4 above)
Comment (pruan-rht) [reply features/online/build.feature:68]: added case id for scenario outline

File: features/networking/sriov.feature (hunk 3 above)
Comment (pruan-rht) [reply features/networking/sriov.feature:524]: added case id for scenario outline

File: features/online/build.feature [hunk 6]
```diff
@@ -106,10 +106,10 @@ Feature: ONLY ONLINE related feature's scripts in this file
...
-      | jboss-eap70-openshift | 1.6       | https://github.com/jboss-developer/jboss-eap-quickstarts.git#7.0.0.GA | kitchensink |
-      | jboss-eap70-openshift | 1.7       | https://github.com/jboss-developer/jboss-eap-quickstarts.git#7.0.0.GA | kitchensink |
+      | case_id   | image_name            | image_tag | source_url                                                            | context_dir |
+      | OCP-20742 | jboss-eap70-openshift | 1.3       | https://github.com/jboss-developer/jboss-eap-quickstarts.git#7.0.0.GA | kitchensink |
+      | OCP-20742 | jboss-eap70-openshift | 1.4       | https://github.com/jboss-developer/jboss-eap-quickstarts.git#7.0.0.GA | kitchensink |
+      | OCP-20742 | jboss-eap70-openshift | 1.5       | https://github.com/jboss-developer/jboss-eap-quickstarts.git#7.0.0.GA | kitchensink |
+      | OCP-20742 | jboss-eap70-openshift | 1.6       | https://github.com/jboss-developer/jboss-eap-quickstarts.git#7.0.0.GA | kitchensink |
+      | OCP-20742 | jboss-eap70-openshift | 1.7       | https://github.com/jboss-developer/jboss-eap-quickstarts.git#7.0.0.GA | kitchensink |
```
[the remark this answers is not in the document]
Comment (pruan-rht) [reply features/online/build.feature:114]: added case id for scenario outline

**Comment by pruan-rht** [comment]
superseded by https://github.com/openshift/verification-tests/pull/2946

**Pull request closed without merging**